test/jdk/tools/jlink/JLinkTest.java

```diff
@@ -358,6 +358,39 @@ public static void main(String[] args) throws Exception {
             helper.generateDefaultImage(userOptions, moduleName).assertFailure("Error: Invalid compression level invalid");
         }
 
+        // short command  without argument- JDK-8321139
```

Comment:
We don't usually include JBS bug ids in code comments. For tests which are related to a bug fix, like this one, we include the bug id in the `@bug` tag of the test definition https://openjdk.org/jtreg/tag-spec.html.

This `JLinkTest`, unfortunately has a pre-existing problem that it uses `@bug` more than once (which as per the `@bug` tag specification, isn't correct). So you will have to do some extra work with this one and address that part too. Ultimately, for this test, the `@bug` should now be:

```
@bug 8189777 8194922 8206962 8240349 8321139
```
While at it, please also update the `@bug` line in `TaskHelperTest` to include 8321139.

Reply:
Good Morning,

For `JLinkTest `, I noticed more bug IDs among other comments so I added those to `@bug`. I added the new bug ID to `TaskHelperTest `. Please let me know if there is some other comment that I should modify/remove.

Reply:
Hello Ana, thank you for spotting that and addressing it. I verified those bug ids and the update you have here looks good to me.